pypeit/core/skysub.py

```diff
@@ -223,27 +223,50 @@ def global_skysub(image, ivar, tilts, thismask, slit_left, slit_righ, inmask=Non
 
 
 
-# TODO -- This needs JFH docs, desperately
-def skyoptimal(wave, data, ivar, oprof, sortpix, sigrej=3.0, npoly=1, spatial=None, fullbkpt=None):
+def skyoptimal(piximg, data, ivar, oprof, sigrej=3.0, npoly=1, spatial_img=None, fullbkpt=None):
     """
-    Utility routine used by local_bg_subtraction_slit
+    Utility routine used by local_skysub_extract that performs the joint b-spline fit for sky-background
+    and object flux.
 
     Args:
-        wave:
-        data:
+        piximg `numpy.ndarray`_:
```

Comment:
we are missing () around the object types
  here and throughout

Reply:
Added.